docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -468,7 +468,14 @@ You can also cancel all pending tasks at once using the [runner.stop](#stop) fun
 The quarantine mode is designed to isolate *non-deterministic* tests (that is, tests that sometimes pass and fail without any apparent reason)
 from the rest of the test base (*healthy* tests).
 
-In this mode, a failed test is executed several times. The test result depends on the outcome (passed or failed) that occurs most often. That is, if the test fails on most attempts, the result is failed. If the test result differs between test runs, the test is marked as unstable.
+When the quarantine mode is enabled, tests run according to the following logic:
+
+1. A test runs at the first time. If it passes, TestCafe proceeds to the next test. If the test fails, it enters the quarantine mode.
+2. The test runs over and over until it shows the same result (passed or failed) three times in total.
```

Comment:
IMO, "until it fails or passes three times" looks simpler.

Reply:
This was my first version, but I thought that it could be understood as "runs three times".